spec/features/admin/sign_in_spec.rb

```diff
@@ -18,6 +18,27 @@
     expect(current_path).to eq '/account'
   end
 
+  context 'localized' do
```

Comment:
context description is vague

Reply:
@wjwitek can you expand this context description, i.e. something like "with locale set to fr"

Reply:
oops nevermind